File: comms/bulk-messaging/messages/send-message/send-message.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.messages.send(
    to=[
        {
            "address": "+14153902337",
            "channel": "PHONE"
        }
    ],
    content={
        "text": "Hello, World!"
    },
)

File: comms/bulk-messaging/messages/send-message/send-message.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.messages.send({
        to: [
            {
                address: "+14153902337",
                channel: "PHONE",
            },
        ],
        content: {
            text: "Hello, World!",
        },
    });
}
main();
